react/components/molecules/date-input/date-input.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, boolean, number, text } from "@storybook/addon-knobs";
+
+import { DateInput } from "./date-input";
+
+storiesOf("Components/Molecules/DateInput", module)
+    .addDecorator(withKnobs)
+    .add("DateInput", () => {
+        const header = text("Header", "Birth Date");
+        const disabled = boolean("Disabled", false);
+        const activeOpacity = number("Active Opacity", -1);
+
+        return (
+            <DateInput
+                value={new Date()}

```

Comment:
Working with `Date` as an actual value seems tricky since the object is mutable. Consider this scenario:
- You have a date `x`
- You pass that as a value
- You do `x.setDate`

I believe that this would "break" the component as it wouldn't react to the value change.

Long story short: wouldn't it be safer to just use an integer representing the epoch? I belive most our util functions do it.